### PR TITLE
Added Long-Term Benefit Cards

### PR DESCRIPTION
**Pull request by mexi-cano**
What (if any) features are you implementing?
- Created component for long-term benefits cards

Screenshot 2024-01-30 at 11 34 32 AM

Screenshot 2024-01-30 at 11 34 19 AM

### CODE REVIEW

File: src/Components/Results/Programs/ProgramCard.tsx [hunk 1]
```diff
@@ -9,10 +10,26 @@ type ProgramCardProps = {
...
+  const estimatedAppTime = program.estimated_application_time;
+  const estimatedSavings = program.estimated_value;
+
   return (
-    <div>
-      <ResultsTranslate translation={program.name} />
-      <Link to={`/${uuid}/results/benefits/${program.program_id}`}>Apply</Link>
+    <div className="result-program-container">
+      <div className="result-program-more-info">
+        <ResultsTranslate translation={program.name} />
```
Comment (CalebPena) [inline src/Components/Results/Programs/ProgramCard.tsx:19]: You assign `estimatedAppTime` and `estimatedSavings` to variables, but you inline `program.name`. Could you either move `program.name` to it's own variable, or inline the other variables.

File: src/Components/Results/Results.css
Comment (CalebPena) [inline src/Components/Results/Results.css:1]: This works fine. Another way that you could have done this is with `display: grid` but no need to change it.

**Review by CalebPena — APPROVED** [review]
Looks good.

**Comment by CalebPena** [comment]
Can you pull from `feat/results_redesign`

**Comment by sydney-devine** [comment]
> What (if any) features are you implementing?
> 
> * Created component for long-term benefits cards
> 
> Screenshot 2024-01-30 at 11 34 32 AM Screenshot 2024-01-30 at 11 34 19 AM

Any way on the mobile view to get "Application Time" on one line? 

Also, need to change "Savings" to be "Estimated Savings"

File: src/Components/Results/Programs/ProgramCard.tsx (hunk 1 above)
Comment (mexi-cano) [reply src/Components/Results/Programs/ProgramCard.tsx]: Updated. Thanks!

**Comment by mexi-cano** [comment]
> Any way on the mobile view to get "Application Time" on one line?
> 
> Also, need to change "Savings" to be "Estimated Savings"

Updated. 

Screenshot 2024-01-30 at 2 25 42 PM

**Comment by sydney-devine** [comment]
> > Any way on the mobile view to get "Application Time" on one line?
> > Also, need to change "Savings" to be "Estimated Savings"
> 
> Updated.
> 
> Screenshot 2024-01-30 at 2 25 42 PM

Sorry last thing I swear - add a colon after "savings" (Estimated Savings:) just so it matches Application Time. 

These changes look great - TY so much for the work on this!

**Comment by mexi-cano** [comment]
> > > Any way on the mobile view to get "Application Time" on one line?
> > > Also, need to change "Savings" to be "Estimated Savings"
> > 
> > 
> > Updated.
> > Screenshot 2024-01-30 at 2 25 42 PM
> 
> Sorry last thing I swear - add a colon after "savings" (Estimated Savings:) just so it matches Application Time.
> 
> These changes look great - TY so much for the work on this!

Added! Thank you!

Screenshot 2024-01-31 at 11 26 10 AM

**Review by sydney-devine — APPROVED** [review]
Visual looks good on my end - TY!

**Pull request merged**